### PR TITLE
internal review v1

### PR DESCRIPTION
**Pull request by ryansroberts**
changed from manual to electronic pp

### CODE REVIEW

File: qualitystandards/qsd95/st1/Statement.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ number: 1
 # Quality statement 1: Pulse palpation
 
 ## Quality statement
-Adults with risk factors for atrial fibrillation have a manual pulse palpation.
+Adults with risk factors for atrial fibrillation have a electronic pulse palpation.
```
Comment (simonhambly) [inline qualitystandards/qsd95/st1/Statement.md]: Please make this change throughout

Comment (alidesi) [inline qualitystandards/qsd95/st1/Statement.md]: OK

Comment (andychrismitchell) [inline qualitystandards/qsd95/st1/Statement.md]: No

**Comment by alidesi** [comment]
Comment with a link https://github.com/andychrismitchell/ld-content-test/blob/master/qualitystandards/qsd95/st5/Statement.md#structure

**Pull request closed without merging**